suite-native/receive/src/components/VerificationWalkthroughBottomSheet.tsx

```diff
@@ -14,6 +15,16 @@ export const isVerificationWalkthroughEnabledAtom = atomWithUnecryptedStorage<bo
     true,
 );
 
+const imageStyle = prepareNativeStyle(() => ({
+    width: '100%',
+    height: 322,
```

Comment:
Is it good idea to have absolute heigh here? What about iPads and other big screens?

Reply:
What if screen is too small? I don't like that there are both values defined, why is there width: 100%?

Reply:
updated in [cf98b19](https://github.com/trezor/trezor-suite/pull/10204/commits/cf98b1930ba26d657ef2ad9c733c3628374e8615)